web/src/features/charts/ChartTitle.tsx

```diff
@@ -7,9 +7,10 @@ type Props = {
   translationKey: string;
   unit?: string;
   badge?: React.ReactElement;
+  id: string;
```

Comment:
Could name this with a bit more info

Reply:
Sure, I can update it with `chartId: Charts`